cpp/ql/src/semmle/code/cpp/ir/implementation/aliased_ssa/Instruction.qll

```diff
@@ -268,6 +268,16 @@ module InstructionSanity {
   }
 }
 
+query predicate switchInstructionWithoutDefaultEdge(
+  SwitchInstruction switchInstr, string message, IRFunction func, string funcText
+) {
+  not exists(switchInstr.getDefaultSuccessor()) and
+  message =
+    "SwitchInstruction " + switchInstr.toString() + " without a DefaultEdge in function '$@'." and
+  func = switchInstr.getEnclosingIRFunction() and
+  funcText = Language::getIdentityString(func.getFunction())
+}
+
```

Comment:
This query predicate needs to be moved two lines up so it'll be inside `module InstructionSanity` (and indented). Otherwise you get those extra headings in random test output files.

Reply:
Ah! That module had escaped me. Thanks!